test/new/db/cmd/types

```diff
@@ -1259,7 +1259,7 @@ af
 aaft
 afv
 EXPECT=<<RUN
-var int32_t var_0h @ sp+0x0
+var int32_t var_1454h @ sp+0x0
```

Comment:
uhm is this correct ?

Reply:
Stack grows to that size:

![image](https://user-images.githubusercontent.com/23562137/70443953-ded2aa80-1a77-11ea-9aef-0ab6016eaf60.png)